### ISSUE TITLE
Moving sometimes has strange side effects

### ISSUE THREAD

**Issue by pboos** [zadam/trilium#375]
Somehow in some cases when I move nodes with children, the children get moved outside of that node as well. When I try to reproduce it, first it doesn't happen. But after creating more and moving things around a couple of times it starts happening.

------------------------------------------

Version: 
"appVersion": "0.28.2",
"dbVersion": 123,
"syncVersion": 4,
"buildDate": "2019-01-21T22:55:12+01:00",
"buildRevision": "c654172d33eed3773b5f76074a2c97479b522920",

Running on a server which has quite a distance to my home (ping of 300-400ms)

Log: [log.txt](https://github.com/zadam/trilium/files/2785446/log.txt)

**Comment by zadam** [comment]
Hmm, is it possible that the demo gif is too short? Because what I see is that move of "test31" into "test3" didn't do anything ... or possibly gif ended at that exact moment.

Anyway not sure if this is related, but Trilium's drag & drop behavior was a little bit confusing in that when you started dragging it dragged all selected items. If you didn't click to activate the note before dragging, then selected was both the note you started dragging and the active note (which might have been the child?). I just changed the behavior so dragging multiple notes happens only when CTRL is being pressed.

**Comment by pboos** [comment]
Sorry about that. Seems like the created gif was wrong. I removed it now.

I think what happened was exactly what you said. When I play around with that in mind, it happens the way it you said.

I think the change you made now to make it only this way when CTRL is pressed is the right way. I think people only expect to have the one moved that they are drag/dropping (and not the one as well that is selected). So having CTRL for that feature for those who know what they are doing, is a good idea :).

Looking forward to the new version :). Closing this issue now.
